api/src/services/bctw-service.ts

```diff
@@ -255,6 +288,42 @@ export class BctwService {
     return this._makeGetRequest(HEALTH_ENDPOINT);
   }
 
+  /**
+   * Upload a single or multiple zipped keyX files to the BCTW API.
+   *
+   * @param {Express.Multer.File} keyX
+   * @return {*}  {Promise<string>}
+   * @memberof BctwService
+   */
+  async uploadKeyX(keyX: Express.Multer.File) {
+    const formData = new FormData();
+    formData.append('xml', keyX.buffer, keyX.originalname);
+    const config = {
+      headers: {
+        ...formData.getHeaders()
+      }
+    };
+    const response = await this.axiosInstance.post(UPLOAD_KEYX_ENDPOINT, formData, config);
+    const data: IUploadKeyxResponse = response.data;
+    if (data.errors.length) {
+      const actualErrors: string[] = [];
+      for (const error of data.errors) {
+        // Ignore errors that indicate that a keyX already exists
```

Comment:
Just curious why we ignore this?

Reply:
The reason for this is because it won't effect the end result. Basically, if the user adds a keyx file, but it already exists in BCTW, there is no harm done because the BCTW is able to retrieve telemetry.